### ISSUE TITLE
Kdyby/DateTimeProvider

### ISSUE THREAD

**Issue by enumag** [GSokol/DateTimeProvider#1]
Not sure if you're aware but there is a similar project here:

https://github.com/Kdyby/DateTimeProvider
https://github.com/Kdyby/DateTimeProviderBundle

**Comment by GSokol** [comment]
Yes, that's looks very similar... I'm going to check it more carefully and, if it's true, migrate my projects to that one and delete this repo. Thank you!

**Comment by enumag** [comment]
Cool. Let us know if you have any suggestions for improvements based on your repository. Thanks!
